dagger/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+
+	"dagger/embedded-cluster/internal/dagger"
+)
+
+// dagger call build-operator-package --ec-version test export --path build/operator-package
```

Comment:
if you relocate this to your BuildOperatorPackage it'll show up in the doc/help strings on the cli.